.github/workflows/generate_workflows_lib/src/generate_workflows_lib/test.yml.j2

```diff
@@ -28,6 +28,9 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: "{{ job_data.python_version }}"
+          {%- if job_data.python_version == "3.13" %}
```

Comment:
It looks like we can drop this

Reply:
Yes. I think we can drop this. @antonpirker are you open to continue the fixes here?